htp/htp_config_private.h

```diff
@@ -238,6 +238,12 @@ struct htp_cfg_t {
     htp_hook_t *hook_request_body_data;
 
     /**
+     * Receives raw request body data, starting immediatelly after empty line
+     * that terminates list of request headers.
+     */
+    htp_hook_t *hook_request_raw_body_data;
```

Comment:
Even thought it's in a file named 'private' it actually exposed. At least Suricata uses it, and even though we never seem to access the structure directly, I am rather safe that sorry. When we bump to 0.6.x we can reshuffle it again. A similar case is here: https://github.com/OISF/libhtp/commit/5b43affd3c9bc1c4280a54266a92f6d0ad7600d1#diff-9815a4e5141bd3d3914c1b5f9295fdecR337